packages/cosmic-swingset/test/export-storage.test.js

```diff
@@ -48,7 +48,11 @@ const makeBatchChainStorage = published => {
         return entries(path).map(([key]) => key);
       }
       case 'setWithoutNotify':
```

Comment:
For consistency with the other `switch` cases:
```suggestion
      case 'setWithoutNotify': {
```